src/app/components/content-page/content-page.component.scss

```diff
@@ -12,6 +12,11 @@ main {
   grid-template-columns: auto auto auto;
 }
 
+.img-button {
+  background-color: rgba(0, 0, 0, 0);
+  border: none;
```

Comment:
Can these be done with bootstrap?

Reply:
i changed the border to bootstrap the rest i dont think can be done with bootstrap